.github/workflows/lint-code.yml

```diff
@@ -58,3 +58,23 @@ jobs:
             Thanks again for your contribution!
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           allow-repeats: false
+
+  isort:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source-code repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install isort
+        run: python -m pip install isort
+
```

Comment:
Add isort to the `requirements-dev.txt` instead.

```suggestion

```

Reply:
I am not sure about that, isort is not required to run the tools, the same as prettier or black